### ISSUE TITLE
Registration page does not allow choosing status "Other"

### ISSUE THREAD

**Issue by maverickwoo** [picoCTF/picoCTF#146]
With the current master c2f4dce, selecting "Other" as the status in the registration page will result in "Instructor" being selected.

Tested on Chrome 67 and FireFox 61.

**Comment by hiliang-cmu** [comment]
The way it's currently setup, the selects are:

```
<option value="eligible">Competitor</option>
<option value="ineligible">Instructor</option>
<option value="ineligible">Other</option>
```
With `Instructor` and `Other` sharing the same option for `value`. I don't know specifically which mechanism it is, but something in React has a listener for whenever that changes, and does some sort of data validation or rendering update, which defers to the first instance of option with `ineligible` value.

As experiment, changing the `value` of `Other`  to `eligible` makes it select `Competitor` instead.

Practically though, there is no difference in selecting` Instructor` or `Other`, as only the `eligible` or `ineligible` data gets passed.

In efforts to allow for more useful iser type data, this select set will soon be altered to the actual values (e.g. `high school student`, `college student`, `teacher`, `other`) then left to the server api to save and determine eligibility.

**Comment by dmartin** [comment]
Unsure when resolved (likely #255), but this behavior no longer occurs as of 6b07ba304f3137eb57b9668c0bab4b4caa8129fa
